packages/core/content-manager/admin/src/pages/EditView/components/InputRenderer.tsx

```diff
@@ -41,8 +40,9 @@ type InputRendererProps = DistributiveOmit<EditFieldLayout, 'size'> & {
  */
 const InputRenderer = ({ visible, hint: providedHint, document, ...props }: InputRendererProps) => {
   const { model: rootModel } = useDoc();
+  const docLayout = useDocLayout();
   const documentLayout = useDocumentLayout(document.schema?.uid ?? rootModel);
-  const components = documentLayout.edit.components;
+  const components = { ...documentLayout.edit.components, ...docLayout.edit.components };
```

Comment:
I'm a bit confused by this. Doesn't it mean that we're overwriting the layout specific to the content type in a modal with the layout of the content type from the edit view page underneath?

I also feel like documentLayout will _always_ be overwritten by docLayout, making it not necessary

Reply:
so you mean I can reverse the order of the merging?

Reply:
we did something similar in the Dynamic component btw https://github.com/strapi/strapi/pull/22997/commits/4bbf30d43befd671c5a6201ce6afedc6b9a38fef

Reply:
Thanks for the link, it makes it clearer for me. Could you do the same thing as over there when it comes to variable names? and the order shouldn't matter in the end

Reply:
done